src/layoutcomponent/LandingPageCategories.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styles from "../styles/LandingPage.module.css";
```

Comment:
Follow consistency, if you are using css modules then use css modules for all styles. I see that there are some normal css files and this one is css module file

Reply:
just for particular component style i wanted to create module .i dont want that particular component css to be global